src/go/rpk/pkg/cli/profile/create.go

```diff
@@ -53,7 +62,7 @@ name "default" is used.
   latter corresponds to the path kafka_api.tls.enabled in the profile's YAML.
 
 The --set flag is always applied last and can be used to set additional fields
-in tandem with --from-simple or --from-cloud.
+in tandem with --from-redpanda or --from-cloud.
```

Comment:
why this change?

Reply:
`from-simple` means nothing -- what's simple? `--from-redpanda` at least in a way implies from redpanda.yaml

Reply:
The help text is a bit more explanatory here:
```
      --from-redpanda string   Create and switch to a new profile from a redpanda.yaml file```
I don't think there's a really good / clear name. You can have a local profile saved, so from-local-config could mean the profile itself.